src/com/twitter/intellij/pants/execution/SetJarManifestCommandLineShortener.scala

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package com.twitter.intellij.pants.execution;
+
+import com.intellij.execution.{ConfigurationWithCommandLineShortener, RunManagerListener, RunnerAndConfigurationSettings, ShortenCommandLine}
+
+class SetJarManifestCommandLineShortener extends RunManagerListener {
+  override def runConfigurationAdded( settings: RunnerAndConfigurationSettings) {
+    settings.getConfiguration match {
+      case configuration: ConfigurationWithCommandLineShortener => configuration.setShortenCommandLine(ShortenCommandLine.MANIFEST)
+      case _ =>
+    }
+  }
+}
```

Comment:
Given pants plugin doesn't bring scala runtime into intellij but relies only on intellij's scala plugin, so if scala plugin isn't enabled, it will likely explode here. Do you think it would be okay to rewrite this java?

If this is only for running scala main class, we can move the meta info `resources//META-INF/pants-scala.xml` and keep this scala.

Reply:
Right, thanks!